sprint9/Forms/index.html

```diff
@@ -10,7 +10,7 @@
 <body>
```

Comment:
![image](https://user-images.githubusercontent.com/5993344/69707037-40e6f380-10f9-11ea-8e1e-62f1e9821e57.png)
Takie coś się dzieje jak kilkam w `input`. to błąd.

Reply:
poprawiłam